integration/simulation/utils.go

```diff
@@ -46,15 +46,16 @@ func minMax(arr []uint64) (min uint64, max uint64) {
 }
 
 // Uses the client to retrieve the current rollup head.
-func getHeadRollupHeader(client *obsclient.ObsClient, nodeIdx int) *common.BatchHeader {
+func getHeadRollupHeader(client *obsclient.ObsClient) *common.BatchHeader {
 	headRollupHeight, err := client.RollupNumber()
 	if err != nil {
 		panic(fmt.Errorf("simulation failed due to failed attempt to retrieve head rollup height. Cause: %w", err))
 	}
 
 	headRollupHeader, err := client.RollupHeaderByNumber(big.NewInt(int64(headRollupHeight)))
 	if err != nil {
-		panic(fmt.Errorf("node %d: simulation failed due to failed attempt to retrieve rollup with height %d. Cause: %w", nodeIdx, headRollupHeight, err))
+		// panic(fmt.Errorf("node %d: simulation failed due to failed attempt to retrieve rollup with height %d. Cause: %w", nodeIdx, headRollupHeight, err))
```

Comment:
is this commented on purpose?

Reply:
Yes, it was panicking out of the tests rather than letting them say test.Fail() and give reasons (which was hiding how many nodes are failing and other bits)

Should probably leave a todo and figure out how to properly do the panic?